GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SubjectViewer.xaml.cs

```diff
@@ -1,15 +1,37 @@
-﻿using System.Windows.Controls;
+﻿using GalaxyZooTouchTable.Models;
+using GalaxyZooTouchTable.ViewModels;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
 
 namespace GalaxyZooTouchTable
 {
     /// <summary>
     /// Interaction logic for SubjectViewer.xaml
     /// </summary>
-    public partial class SubjectViewer : UserControl
+    public partial class SubjectViewer : UserControl, IDroppableArea
     {
         public SubjectViewer()
         {
             InitializeComponent();
         }
+
+        public bool IsUnder(Point p)
+        {
+            throw new System.NotImplementedException();
+        }
```

Comment:
I suspect you're forced to implement this. Is that accurate?

Reply:
Yes, this is required because `SubjectViewer` inherits from the `IDroppableInterface`, even though the `IsUnder` method isn't used here. I may adjust this in the future, but it's worth keeping around as a reminder if I plan to restructure this later to truly implement the `IsUnder` function.